src/actions.js

```diff
@@ -93,6 +93,55 @@ module.exports = {
 			},
 		}
 
+		actions.set_active_app = {
+			name: 'Launch App',
+			options: [
+				{
+					type: 'dropdown',
+					label: 'Type',
+					id: 'type',
+					choices: [
+						{ id: 'activity', label: 'Android Activity' },
+						{ id: 'url', label: 'URL' },
+						{ id: 'manifest', label: 'Manifest' },
+						{ id: 'auid', label: 'Application ID' },
+					],
+				},
+				{
+					type: 'textinput',
+					label: 'Value',
+					id: 'app_value',
+					default: '',
+					required: true,
+				},
+			],
+			callback: async function (action) {
+				let opt = action.options
+				const value = encodeURI(opt.app_value.trim())
+				let params = {
+					uri: opt.type === 'activity' ? '' : 'localapp://webappruntime?',
+				}
+				if (opt.type === 'url') {
+					params.uri += 'url=' + value
+				} else if (opt.type === 'manifest') {
+					params.uri += 'manifest_url=' + value
+				} else if (opt.type === 'auid') {
+					params.uri += 'auid=' + value
+				} else if (opt.type === 'activity') {
+					params.uri += value
+				}
+				self.sendCommand('appControl', 'setActiveApp', params)
+			},
+		}
+
+		actions.terminate_apps = {
+			name: 'Terminate Apps',
```

Comment:
Assuming it will close any open apps:
```suggestion
			name: 'Terminate All Apps',
```